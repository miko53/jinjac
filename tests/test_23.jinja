TEST with user callback
display {{ @name }}
a value {{ @speed }} and another one {{ @count }}
now array data
{% for i in @array_data_int -%}
int data: {{ i }}
{% endfor %}
{% for i in @array_data_dbl -%}
int data: {{ i }}
{% endfor %}
{% for i in @array_data_string -%}
int data: {{ i }}
{% endfor %}
-- end test --
