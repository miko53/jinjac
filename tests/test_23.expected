TEST with user callback
display Tyrion
a value 125.580000 and another one 5684
now array data
int data: 5
int data: 8
int data: -159
int data: 68
int data: 156

int data: 2.000000
int data: 598.100000
int data: 14.890000
int data: 144.117000
int data: 0.215000
int data: -159.178000
int data: 0.000000
int data: 157.000000
int data: 0.129840
int data: 159618.190000

int data: good
int data: bad
int data: ugly

-- end test --
